### PR TITLE
Fixes #2012 - issue-list / fixed search bar

### LINKED ISSUE #2012: Refactor: list-issue/
Refactor following templates with the new semantics:

- [x] templates/list-issue.html
- [ ] templates/list-issue/issuelist-search.html
- [x] templates/list-issue/dropdown.html
- [x] templates/list-issue/search-issue-filter.html
- [x] templates/list-issue/search-issue-sorting.html
- [x] templates/list-issue/issuelist-issue.jst
- [x] templates/list-issue/issuelist-sorting.jst
- [x] templates/list-issue/search-issue-search-form.html (REMOVED)

### PR DESCRIPTION
**Pull request by zoepage**
r?  @miketaylr

### CODE REVIEW

File: webcompat/templates/list-issue/issuelist-search.jst [hunk 1]
```diff
@@ -1,20 +1,10 @@
...
-      <button role="button" class="nav-button nav-link js-SearchForm-button" title="Search for an item.">
-        <svg class="icon nav-icon" viewBox="0 0 30 30" role="presentation">
-            <use href="#svg-search2">Search</use>
-        </svg>
-    </button>
---->
   <form id="x-search-bar" method="get" action="/issues">
         <input class="form-field text-field" name="q" placeholder="Search issues by keyword"/>
-        <button class="nav-button js-SearchBarClose" aria-hidden="true" type="button">
+        <button role="button" class="nav-button nav-link js-SearchForm-button" title="Search for an item.">
```
Comment (miketaylr) [inline webcompat/templates/list-issue/issuelist-search.jst]: Why are we removing `type=button`? Does that change behavior to do a form submit?

Comment (zoepage) [reply webcompat/templates/list-issue/issuelist-search.jst]: this definitely looks like a mistake. will fix.

**Comment by miketaylr** [comment]
lgtm, thanks.

**Pull request merged**